lib/core/src/lib/datatable/data/object-datatable-adapter.ts

```diff
@@ -54,9 +57,10 @@ export class ObjectDataTableAdapter implements DataTableAdapter {
         return schema;
     }
 
-    constructor(data: any[] = [], schema: DataColumn[] = []) {
+    constructor(data: any[] = [], schema: DataColumn[] = [], sortingMode: SortingMode = 'client') {
         this._rows = [];
         this._columns = [];
+        this._sortingMode = sortingMode?.toString().toLowerCase() === 'server' ? 'server' : 'client';
```

Comment:
nit: toString() probably isn't necessary

Reply:
Just trying to be super careful incase `sortingMode` is not provided as a string. You think that's overkill?

Reply:
It doesn't really hurt anything, but I would expect all consumers of ObjectDataTableAdapter to be using TypeScript typings, and not need the additional javascript safety checks.  I spot checked some files and didn't see anywhere else where typings weren't trusted, but I could have easily missed another file that does.

Personally, I would just account for sortingMode being undefined, but if it is defined assume its a SortingMode.

Reply:
Sounds good, makes sense to me. I'll update it.